WarOfEmpires.Models/Empires/TroopModel.cs

```diff
@@ -36,5 +39,8 @@ public sealed class TroopModel {
         public bool HasUpkeepRunOut { get; set; }
         public bool HasSoldierShortage { get; set; }
         public string Command { get; set; }
+        public int CurrentStamina { get; set; }
+        public int StaminaToFull { get; set; }
+        public int HealMaxAfford { get; set; }
```

Comment:
This one can probably also have a better name once you use it?

Reply:
Added TODO to rename